masterlist.yaml

```diff
@@ -18417,7 +18422,21 @@ plugins:
     clean:
       - crc: 0x03ED123A
         util: 'SSEEdit v4.0.3'
-        
+
+  ### Even More Khajiit Speak Patches ###
+  - name: 'Chr_KhajiitSpeak.*\.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/40594/' ]
+    after:
+      - 'BA_KhajiitSpeakRedux_MAIN.esp'
+      - 'mjhKhajiitSpeak.esp'
+
+  ### Khajiit Speak Extended ###
+  - name: 'Khajiit Speak Extended SSE.*\.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/18441/' ]
+    after:
+      - 'BA_KhajiitSpeakRedux_MAIN.esp'
+      - 'mjhKhajiitSpeak.esp'
```

Comment:
You could put all the KS patches together.

```
  ### Khajiit Speak Patches ###
  - name: 'Chr_KhajiitSpeak.*\.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/40594/'
        name: 'Even More Khajiit Speak Patches'
    after:
      - 'BA_KhajiitSpeakRedux_MAIN.esp'
      - 'mjhKhajiitSpeak.esp'

  - name: 'Khajiit Speak \+.*\.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/35294/'
        name: 'Raptor''s Khajiit Speak Patch Collection'
    after:
      - 'BA_KhajiitSpeakRedux_MAIN.esp'
      - 'mjhKhajiitSpeak.esp'

  - name: 'Khajiit Speak Extended SSE.*\.esp'
    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/18441/' ]
    after:
      - 'BA_KhajiitSpeakRedux_MAIN.esp'
      - 'mjhKhajiitSpeak.esp'
```

Reply:
I just separated them since they're all on different pages, but that's reasonable too.